tools/internal_ci/windows/bazel_rbe.bat

```diff
@@ -0,0 +1,18 @@
+@rem Copyright 2019 gRPC authors.
+@rem
+@rem Licensed under the Apache License, Version 2.0 (the "License");
+@rem you may not use this file except in compliance with the License.
+@rem You may obtain a copy of the License at
+@rem
+@rem     http://www.apache.org/licenses/LICENSE-2.0
+@rem
+@rem Unless required by applicable law or agreed to in writing, software
+@rem distributed under the License is distributed on an "AS IS" BASIS,
+@rem WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+@rem See the License for the specific language governing permissions and
+@rem limitations under the License.
+
+choco install bazel -y --version 0.23.2
+cd github/grpc
+set PATH=%PATH%;C:\python27\
+bazel --bazelrc=tools/remote_build/windows.bazelrc build :all --incompatible_disallow_filetype=false --google_credentials=%KOKORO_GFILE_DIR%/rbe-windows-credentials.json
```

Comment:
future improvements: I think in order to get a "bazel" link for the build in Fusion UI, you'll need to do something like this: https://github.com/grpc/grpc/blob/d5769552c6db5dea7c6c7c4e3d8aa6a0ab5368e8/tools/internal_ci/linux/grpc_bazel_on_foundry_base.sh#L37

Reply:
ack